documentation/compose/kafka-ui-connectors-auth.yaml

```diff
@@ -105,47 +98,17 @@ services:
       CONNECT_REST_EXTENSION_CLASSES: "org.apache.kafka.connect.rest.basic.auth.extension.BasicAuthSecurityRestExtension"
       KAFKA_OPTS: "-Djava.security.auth.login.config=/conf/kafka_connect.jaas"
 
-#      AWS_ACCESS_KEY_ID: ""
-#      AWS_SECRET_ACCESS_KEY: ""
+  #      AWS_ACCESS_KEY_ID: ""
+  #      AWS_SECRET_ACCESS_KEY: ""
 
   kafka-init-topics:
-    image: confluentinc/cp-kafka:5.3.1
+    image: confluentinc/cp-kafka:7.2.1
     volumes:
       - ./message.json:/data/message.json
-    depends_on:
-      - kafka1
-    command: "bash -c 'echo Waiting for Kafka to be ready... && \
-               cub kafka-ready -b kafka1:29092 1 30 && \
-               kafka-topics --create --topic second.users --partitions 3 --replication-factor 1 --if-not-exists --zookeeper zookeeper1:2181 && \
-               kafka-topics --create --topic second.messages --partitions 2 --replication-factor 1 --if-not-exists --zookeeper zookeeper1:2181 && \
-               kafka-topics --create --topic first.messages --partitions 2 --replication-factor 1 --if-not-exists --zookeeper zookeeper0:2181 && \
-               kafka-console-producer --broker-list kafka1:29092 -topic second.users < /data/message.json'"
-
-  create-connectors:
```

Comment:
where did this container go?

Reply:
Can you be more specific about this?

Reply:
`create-connectors` has been removed as far as I can see

Reply:
When I checked this file, there were many errors in it. Considering this setup is for testing connectors with auth, I remove the irrelevant containers. For `create-connectors`, it can be seen in e2e-tests.yaml. If `create-connectors` is required in this file, I can add it back.

Reply:
aight, it's indeed not much related. Let's keep it this way.